cobigen-cli/cli/src/main/java/com/devonfw/cobigen/cli/commands/GenerateCommand.java

```diff
@@ -333,17 +333,17 @@ public <T extends GenerableArtifact> void generate(Path inputFile, Object input,
     boolean generationSuccess = ValidationUtils.checkGenerationReport(report);
     if (generationSuccess) {
       Set<Path> generatedJavaFiles = report.getGeneratedFiles().stream().filter(e -> e.getFileName().endsWith(".java"))
-              .collect(Collectors.toSet());
+          .collect(Collectors.toSet());
       if (!generatedJavaFiles.isEmpty()) {
         try {
           ParsingUtils.formatJavaSources(generatedJavaFiles);
         } catch (FormatterException e) {
           LOG.warn(
-                  "Generation was successful but we were not able to format your code. Maybe you will see strange formatting.");
+              "Generation was successful but we were not able to format your code. Maybe you will see strange formatting.");
         }
       }
     } else {
-      LOG.error("Generation not successful! For more info execute in verbose mode (-v).");
+      LOG.error("Generation not successful! " + (LOG.isDebugEnabled() ? MessagesConstants.VERBOSE_HINT : ""));
```

Comment:
LOG should not be used with string concatenation. Please use {} instead.

Reply:
Concatenation has been removed in next commit.